packages/ui/src/providers/DocumentInfo/index.tsx

```diff
@@ -339,6 +339,11 @@ const DocumentInfo: React.FC<
               ...versionParams.where,
               and: [
                 ...versionParams.where.and,
+                {
+                  'version._status': {
+                    equals: 'draft',
+                  },
+                },
```

Comment:
Why is this needed? I expected that we want to include published also.

Reply:
This is used for this indicator:
<img width="253" alt="image" src="https://github.com/user-attachments/assets/f95225b8-2012-41e8-9899-69b819533f3c">

https://github.com/payloadcms/payload/blob/067d353cdd2869623a2361322625b685ff355f5e/packages/ui/src/elements/Status/index.tsx#L42-L51   

The logic is that if there are newer versions - this should be be marked as "Changed", I.e currently this doc has status `'published'` but there is a new version that's has status `'draft``, either from "Save Draft" or autosave which creates a draft version.

We relied on a hack in `saveVersion` to have the same `updatedAt` with `doc.updatedAt` on the version itself so then this particular query doesn't return any docs when publishing.
https://github.com/payloadcms/payload/pull/8646/files#diff-e47751f12d8e80d3d391d2d4ab2a7d6739bdaf4adc3755674d045180d2741492L117-L122
But from the database perspective, it's not true that this field should have the same `updatedAt`, it should only apply for `version_updatedAt`. Imagine we updated the row in `posts` table, but the version in `_posts_v` is created 20 seconds later because of some lag.

In also this creates a bad situation for sorting by `updatedAt` in SQL databases because you can have different versions with the same `updatedAt` value.

Reply:
That makes sense. I didn't look at how this was being used and the variables in the component aren't helping.